superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm/CommonParameters.tsx

```diff
@@ -85,10 +86,10 @@ export const databaseField = ({
     value={db?.parameters?.database}
     validationMethods={{ onBlur: getValidation }}
     errorMessage={validationErrors?.database}
-    placeholder={t('e.g. world_population')}
+    placeholder={t(placeholder ?? 'e.g. world_population')}
```

Comment:
I don't have a great sense of how translation strings work, but I wonder if this might break the build step of extracting translatable strings to have the argument of `t()` be dynamically set at render time.  If you or someone else knows for sure definitely correct me, but would it be safer to have this just be `placeholder ?? t('e.g. world_population')` and instead call `t()` for `placeholder` where the string literal is defined in `superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx`?